index.js

```diff
@@ -52,10 +61,162 @@ const config = {
             },
         ],
         curly: ["error"],
+        eqeqeq: ["error", "smart"],
```

Comment:
I typically agree, but I have run into cases where this would have prevented bugs. For example, I've noticed GraphQL type gen tools (zeus) incorrectly typing things as `undefined` when they are actually `null`, so would have been caught by `==`.

Though perhaps it's best to use an ESLint ignore or assertion with a comment in that case, to be explicit and ensure the developer is aware of the implications.

Reply:
https://github.com/graphql-editor/graphql-zeus/issues/262 :(

Reply:
I've reverted this to `always` - I reckon we can tune it down on a project-by-project basis if there are more cases like the Zeus example above